Library/Homebrew/github_runner_matrix.rb

```diff
@@ -53,11 +35,11 @@ def initialize(testing_formulae, deleted_formulae, dependent_matrix:)
     freeze
   end
 
-  sig { returns(T::Array[RunnerSpecHash]) }
+  sig { returns(T::Array[T::Hash[String, T.untyped]]) }
   def active_runner_specs_hash
```

Comment:
```suggestion
  def active_runner_specs
```

Reply:
Will need to be fixed in `dev-cmd/determine-test-runners`.